tests/foreman/api/test_computeresource_libvirt.py

```diff
@@ -119,6 +125,8 @@ def test_positive_create_with_provider(self):
         :expectedresults: Compute resources are created with expected providers
 
         :CaseImportance: Critical
+
+        :CaseLevel: Component
```

Comment:
same as above and below tests where applicable for `CaseLevel` and `CaseImportance`